include/libdnf/module/module_item.hpp

```diff
@@ -165,17 +167,18 @@ class ModuleItem {
     // TODO(jmracek) Read a real decision from the modular solver
     bool is_active() const;
 
-private:
-    friend class ModuleSack;
-    friend class ModuleMetadata;
-    friend ::ModuleTest;
-
     ModuleItem(_ModulemdModuleStream * md_stream, const ModuleSackWeakPtr & module_sack, const std::string & repo_id);
```

Comment:
Should we keep this constructor private?
I believe it is not needed for `ModuleQuery` and I think API users don't have a way to make valid `_ModulemdModuleStream` anyway.

Reply:
It's not very well visible, because of how the changes are shown, but the constructor is not private.

Reply:
Yes, now it is not private, but I think it should be.

Reply:
Oh, sorry, I misunderstood, you are right